src/components/cylc/workflow/Toolbar.vue

```diff
@@ -165,14 +167,12 @@ export default {
     },
     views: [
       {
-        name: 'tree',
-        title: 'Tree',
+        name: TreeView.name,
         icon: mdiFileTree
 
       },
       {
-        name: 'mutations',
-        title: 'Mutations',
+        name: MutationsView.name,
```

Comment:
We don't need the lower-case and upper-case variations. Easier to fire an event just with the name and also use it for displaying the value. Plus, this way we just need to update the `.name` in one place to get the views&components that use it to update too.